src/Microsoft.AspNetCore.Razor.Language/CodeGeneration/CSharpCodeWriter.cs

```diff
@@ -302,9 +362,10 @@ public CSharpCodeWriter WriteEndMethodInvocation(bool endLine)
         }
```

Comment:
Yes. This and `WriteEndMethodInvocation` combined is used in about 50 places. Might be useful. But I don't feel strongly either way.